drivers/project_scorecard/app/views/project_scorecard/warehouse_reports/scorecards/edit.haml

```diff
@@ -0,0 +1,14 @@
+= render 'breadcrumbs'
+
+= simple_form_for @report, url: project_scorecard_warehouse_reports_scorecard_path(@report) do |f|
+
+  .row
+    .col-sm-8
+      %h1 Project Scorecard
+    .col-sm-4
+      .text-right
+        - if @report.status.in?(['pre-filled', 'ready'])
+          = f.submit 'Save & Complete', class: 'btn btn-secondary'
+        = f.submit 'Save Changes', class: 'btn btn-primary'
```

Comment:
Let's include these buttons at the bottom as well, and provide a "Mark Complete" button on the show page near the Edit button.